dockerfiles/Dockerfile.openvino-rhel8

```diff
@@ -0,0 +1,102 @@
+# Build stage
+FROM registry.access.redhat.com/ubi8/ubi:8.4
+
+WORKDIR /code
+
+ARG MY_ROOT=/code
+ARG DEVICE=CPU_FP32
+ARG ONNXRUNTIME_REPO=https://github.com/microsoft/onnxruntime
+ARG ONNXRUNTIME_BRANCH=main
+
+ENV INTEL_OPENVINO_DIR=/opt/intel/openvino_2022.1.0.643
```

Comment:
Lets move to 2022.2 ?

Reply:
Since OV2022.2 official release is not done, waiting on it to update the OV version. 
However, validated the pre release of 2022.2 which works as expected.

Reply:
FIne